service/src/test/java/uk/gov/hmcts/reform/fpl/service/document/transformer/RespondentStatementsTransformerTest.java

```diff
@@ -66,7 +66,11 @@ void shouldReturnRespondentStatementsForHmctsViewAndSortByUploadedDateTime() {
             caseData, DocumentViewType.HMCTS);
 
         assertThat(documentBundleView).isEqualTo(List.of(
-            DocumentBundleView.builder().name("Respondent 1 statements").documents(expectedDocuments).build()));
+            DocumentBundleView.builder()
+                .name(RESPONDENT1.getValue().getParty().getFullName() + " statements")
```

Comment:
You know their full names so just add them to the string instead of doing the method call

Reply:
updated the tests